packages/@react-aria/color/docs/useColorSlider.mdx

```diff
@@ -120,13 +178,19 @@ function ColorSlider(props) {
 
 ### Vertical
 
+This example shows how to build a vertical color slider. The main difference from horizontal color sliders is the addition of the
+`orientation: 'vertical'` option to `useColorSlider`. This automatically adjusts the internal positioning and dragging logic.
+Additionally, the vertical slider does not have a visible label or `<output>` element. This is done by simply not using the returned
```

Comment:
```suggestion
Additionally, this example does not have a visible label or `<output>` element. This can be done by simply not using the returned
```

Reply:
Slider has "Additionally, this example shows how to build a slider without a visible label or output element."